src/components/charts/charts.util.tsx

```diff
@@ -51,11 +51,29 @@ export const ValueContainer = props => {
         count = length
     }
 
+    function FilterName() {
```

Comment:
Don't use uppercase for functions which are not react components.

Reply:
fixed